source/common/http/http1/balsa_parser.cc

```diff
@@ -126,6 +126,13 @@ BalsaParser::BalsaParser(MessageType type, ParserCallbacks* connection, size_t m
     : message_type_(type), connection_(connection) {
   ASSERT(connection_ != nullptr);
 
+  quiche::HttpValidationPolicy http_validation_policy;
```

Comment:
Great use of the struct, very clear!

Reply:
🤞 that we always remember to set a default value in struct otherwise could run into balsa update wherein Envoy side isn't updated here and if no default => uninit value used :( .

Reply:
Yup, good point.  It also helps to be explicit here about which checks are enabled and which ones are disabled.